app/services/application_state_change.rb

```diff
@@ -111,6 +111,12 @@ def self.states_visible_to_provider
     STATES_VISIBLE_TO_PROVIDER - [:interviewing]
   end
 
+  # TAD receive the same applications as providers, but the `rejected` and `declined`
+  # states are more specific when they are rejected or declined by default.
+  def self.states_visible_to_tad
```

Comment:
Should this be part of the scope in `DataAPI::TADExport`?

Reply:
No, that would be a bit weird I think. The states here aren't all real states, rejected_by_default and declined_by_default are kind of virtual, I wouldn't want them in a query perhaps.

Reply:
Got it